src/fmu/datamodels/fmu_results/enums.py

```diff
@@ -7,48 +7,243 @@ class Classification(StrEnum):
+    """Thickness maps (isochores) representing facies thickness.
+
+    Typically provided as an `xtgeo.RegularSurface` for export.
+    """
+
     fault_lines = "fault_lines"
+    """Intersections between fault planes and horizons.
+
+    Typically provided as an `xtgeo.Polygons` for export.
+    """
+
+    fault_surface = "fault_surface"
+    """A surface, triangulated or not, representing a fault.
+
+    Typically provided either as an RMS FaultRoom GeoJSON surface or an fmu-dataio
+    `TSurfData` for export.
+    """
+
     fault_properties = "fault_properties"
+    """Properties, such as permeability and porosity, on a fault.
+
+    Typically provided as a GeoJSON file derived from RMS FaultRoom for export.
+    """
+
     field_outline = "field_outline"
+    """Polygons representing the outline of a field, initial (static) conditions.
+
+    Typically provided as an `xtgeo.Polygons` for export.
+    """
+
     field_region = "field_region"
+    """Delineated or named region within a field.
+
+    Typically provided as an `xtgeo.Polygons` for export.
+    """
+
     fluid_contact = "fluid_contact"
+    """Depth surface representing a fluid contact used per realization, intial (static)
+    conditions.
+
+    Typically provided as an `xtgeo.RegularSurface` for export.
+    """
+
     khproduct = "khproduct"
+    """The product of permeability (k) and reservoir thickness (h).
+
+    Typically provided as an `xtgeo.RegularSurface` for export.
+    """
+
     lift_curves = "lift_curves"
+    """Table representing the relationship between production rates and pressures.
+
+    Typically provided as a Pandas `Dataframe` for export.
+    """
+
     named_area = "named_area"
+    """A named area within a field that is _not_ a region.
+
+    Typically provided as an `xtgeo.Polygons` for export.
+    """
+
     parameters = "parameters"
+    """The ERT parameters generated for the realization.
+
+    Typically provided as a Pandas `Dataframe` for export.
+
+    ```tip
+    You should not export this manually. This is done automatically by
+    [`SUMO_UPLOADER`](https://fmu-sumo-uploader.readthedocs.io/en/latest/) when
+    uploading files.
+    ```
+    """
+
     pinchout = "pinchout"
+    """Polygons designating a pinchout.
+
+    Typically provided as an `xtgeo.Polygons` for export.
+    """
+
     property = "property"
+    """A property, like permeability or porosity, belonging to a 3D grid.
+
+    Typically provided as an `xtgeo.GridProperty`.
+    """
+
     pvt = "pvt"
+    """Tabular pressure-volume-temperature data.
+
+    Typically provided as a Pandas `Dataframe` for export.
+
+    ```tip
+    You should not export this manually. Use
+    [`SIM2SUMO`](https://fmu-sumo-sim2sumo.readthedocs.io/en/latest/).
+    ```
+    """
+
     regions = "regions"
+    """Distinct areas within the field that have different characteristics.
+
+    Examples may be volume regions or contact regions.
+
+    Typically provided as an `xtgeo.Polygons`.
```

Comment:
Could also be a 3D property  (? @tnatt )

Reply:
I think I agree that it could be a grid property. Could technically also be a discrete surface (weird, but doable). I think we need to be more explicit on _type_ of regions in the future. E.g. `volume_regions`, `field_regions` and `custom_region` or something like that.